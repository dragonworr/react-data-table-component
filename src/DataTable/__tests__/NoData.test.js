import 'jest-styled-components';

import React from 'react';
import { shallow } from 'enzyme';

import NoData from '../NoData';

test('component <NoData /> should render correctly when a component is passed that is a string', () => {
  const wrapper = shallow(<NoData component="A Component that is passed in">A String that is passed in</NoData>);
  expect(wrapper).toMatchSnapshot();
});

test('component <NoData /> should render correctly when a component is passed that is a react component', () => {
  const wrapper = shallow(<NoData component={<div>A String that is passed in</div>}><div>A String that is passed in</div></NoData>);
  expect(wrapper).toMatchSnapshot();
});
