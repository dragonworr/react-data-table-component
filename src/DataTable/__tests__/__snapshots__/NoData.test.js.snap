// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component <NoData /> should render correctly when a component is passed that is a react component 1`] = `
<NoData__NoDataWrapperMessage>
  <div>
    A String that is passed in
  </div>
</NoData__NoDataWrapperMessage>
`;

exports[`component <NoData /> should render correctly when a component is passed that is a string 1`] = `
<NoData__NoDataWrapperMessage>
  A Component that is passed in
</NoData__NoDataWrapperMessage>
`;
